### DISCUSSION TITLE
Cannot open Artisan Viewer while roasting (Ver. 2.4.6)

### DISCUSSION THREAD

**Discussion by PhilippSchoenbaechler** [artisan-roaster-scope/artisan#828] [Q&A]
Hello

After update to Artisan 2.4.6 it doesn't seem possible anymore, to record a roast and view an old roast in the Artisan Viewer at the same time.  Can someone tell me if this is somehow possible - like the old versions?

Thank you.

Best wishes
Philipp

**Comment by MAKOMO** [comment]
Still possible. That is the whole point of the ArtisanViewer.

**Reply by PhilippSchoenbaechler** [reply]
I'm sorry, but the viewer cannot open a saved alog file on my installation, if a roast is being recorded at the same time.

And just discovered another new phenomenon with 2.4.6:
After a roast finishes, I click "Off". Artisan saves the recording correctly, but then goes straight back to "On".

**Reply by MAKOMO** [reply]
> I'm sorry, but the viewer cannot open a saved alog file on my installation, if a roast is being recorded at the same time.

Works for me and others.
 
> And just discovered another new phenomenon with 2.4.6: After a roast finishes, I click "Off". Artisan saves the recording correctly, but then goes straight back to "On".

Do you have "Keep ON" ticked under menu Config >> Sampling?

**Reply by PhilippSchoenbaechler** [reply]
`Do you have "Keep ON" ticked under menu Config >> Sampling?`
Yes. Shame on me...

`Works for me and others.`
How? Do I need to create a file reference to open alog files with the viewer? Or what else do I have to do to be able to use the Artisan Viewer during a roast?
If I want to open a file, the viewer appears briefly, but then closes again automatically.

**Comment by MAKOMO** [comment]
It would maybe a good idea to give some more details on your setup. Which OS are you using?

**Reply by PhilippSchoenbaechler** [reply]
I using Windows 10.

I have a new insight...
When I open Artisan 2x manually via the desktop-shortcut - not by clicking directly on an alog file - then the second instance is executed as ArtisanViewer. Then I can also look at old roasts via File->Open.

**Reply by PhilippSchoenbaechler** [reply]
I have found a work-arround.

I have create a Batch-File with command:
`start "" "C:\Program Files\Artisan\artisan.exe"`

Then I opened the "open with"-context on a alog-file and selected the bat file instead of artisan.exe, which should be executed when an alog file is clicked.

It's not perfect, but it's works again...

**Comment by MAKOMO** [comment]
Philipp, we might have identified that issue on Windows in the GUI lib underlying Artisan and found a work around.

Could you please test the continuous build and report if this resolves the issue?
https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous

Thanks!

**Reply by PhilippSchoenbaechler** [reply]
I installed the version. At first glance it looks good.
The Artisan viewer open correctly, even when recording a roast.

But it was just a test. The next live roast will be tomorrow...

**Comment by MAKOMO** [comment]
Thanks for this quick test. Awaiting your report from tomorrows session. THANKS!

**Reply by PhilippSchoenbaechler** [reply]
Hi Marko

Sorry, for the delay.
In the meantime we have created around 20 roasts. The V.2.4.7 works perfectly for us so far.

Thanks very much!

**Reply by MAKOMO** [reply]
Thanks for this report!
